### PR TITLE
[MB-1677] Pull in classes.jar during build

### PR DESCRIPTION
**Pull request by rlepinski**
Copies the classes.jar during build instead of adding it to our repo

### CODE REVIEW

File: android-plugin-lib/build.gradle [hunk 1]
```diff
@@ -24,13 +34,21 @@ android {
...
     }
 }
 
 dependencies {
-    compile files('libs/classes.jar')
+    compile fileTree(dir: 'libs', include: '*.jar')
     compile 'com.urbanairship:urbanairship-sdk:+@aar'
 }
 
+task copyUnityClassJar() {
```
Comment (hcrowell) [inline android-plugin-lib/build.gradle]: super nitpicky: suggest `copyUnityClassesJar` for consistency and easier to find & search?

**Comment by hcrowell** [comment]
:tangerine: after reviewing very minor comment above ^

File: android-plugin-lib/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:amazon="http://schemas.amazon.com/apk/res/android"
```
Comment (hcrowell) [inline android-plugin-lib/src/main/AndroidManifest.xml]: just curious why did you remove amazon?

Comment (rlepinski) [inline android-plugin-lib/src/main/AndroidManifest.xml]: wasnt needed in this particular file

**Comment by hcrowell** [comment]
:tangerine:
